src/add-remove.js

```diff
@@ -0,0 +1,100 @@
+// eslint-disable-next-line import/no-cycle
+import checkboxesEvent, {
+  list, save, fixIndex, setList, remove,
+} from './status-update';
+
+const todoList = document.getElementById('todo-list');
+export const items = document.getElementsByClassName('task-item');
```

Comment:
- [ ] It's not clear how to delete a single to-do item. Consider toggling the 3-dot-icon with the delete icon.

Reply:
Alright, Thanks for the review